ui/src/components/modlog.tsx

```diff
@@ -8,6 +8,7 @@ import { msgOp, addTypeInfo, fetchLimit } from '../utils';
 import { MomentTime } from './moment-time';
```

Comment:
I haven't actually done the i18n work for this modlog file yet, mainly because its a lot of sentences, and I was too tired to do it at the time. At some point I'll get around to it, but for now, revert this file to master.